### ISSUE TITLE
Loading fails on Windows with Ruby 2.2

### ISSUE THREAD

**Issue by ccoenen** [copiousfreetime/hitimes#39]
hitimes-1.2.2-x86-mingw32 fails to load on my machine with ruby 2.2. I have to explicitly install hitimes this way:

```shell
# this works:
gem install hitimes -v 1.2.2 --platform ruby
```

If i don't (that is, if i just let bundler do the installing for me) the resulting error looks like this:

```
C:/Tools/Ruby22/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require': cannot load such file -- hitimes/hitimes (LoadError)
        from C:/Tools/Ruby22/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        from C:/Tools/Ruby22/lib/ruby/gems/2.2.0/gems/hitimes-1.2.2-x86-mingw32/lib/hitimes.rb:37:in `rescue in <top (required)>'
        from C:/Tools/Ruby22/lib/ruby/gems/2.2.0/gems/hitimes-1.2.2-x86-mingw32/lib/hitimes.rb:32:in `<top (required)>'
        from C:/Tools/Ruby22/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:128:in `require'
        from C:/Tools/Ruby22/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:128:in `rescue in require'
        from C:/Tools/Ruby22/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:39:in `require'
        from C:/Tools/Ruby22/lib/ruby/gems/2.2.0/gems/timers-4.0.1/lib/timers/group.rb:4:in `<top (required)>'
```

Hitimes is loadded through celluloid's dependency to the timers gem, I'm not specifying the gem myself.

Is there any way i can help debug it?

(credit for the solution goes to [this StackOverflow Answer](http://stackoverflow.com/questions/28985481/hitimes-require-error-when-running-jekyll-serve-on-windows-8-1).)

**Comment by copiousfreetime** [comment]
No need to debug, this is because I haven't released a fat-binary gem with support for ruby 2.2 yet on windows. It looks like Ruby 2.2.1 is the first release of Ruby 2.2 for windows and was just released on march 6. hitimes 1.2.2 was released in January. 

I've opened issue #40 to track releaseing a new fat binary version of hitimes for ruby 2.2. support on windows.

**Comment by copiousfreetime** [comment]
For folks that find this issue through the Internet, in general please install hitimes `gem install hitimes` When you install using the `--platform ruby` flag, you will install the source code version of the gem, which is not what you want when installing hitimes on windows.
